packages/cozy-harvest-lib/src/components/OAuthWindow.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+import { prepareOAuth, checkOAuthData, terminateOAuth } from '../helpers/oauth'
+import withDomain from './hoc/withDomain'
+import Popup from './Popup'
+
+const OAUTH_POPUP_HEIGHT = 800
+const OAUTH_POPUP_WIDTH = 800
+
+/**
+ * OAuth window is responsible for managing the OAuth process on the client
+ * side, form storing info into local storage, load OAuth window and listen
+ * for data from it.
+ *
+ * The goal of our OAuth Workflow is to retrieve an account id.
+ *
+ * The main part of our OAuth workflow is done by the stack, which redirect
+ * to the OAuth provider authentification endpoint, handle redirection, creates
+ * account and forward the id of the created account.
```

Comment:
```suggestion
 * account and forwards the id of the created account.
```

It is not clear to whom the id of the created account is forwarded.